testapp/testapp/testmain/tests/test_models.py

```diff
@@ -143,6 +143,22 @@ def test_raise_validation(self):
         self.assertEqual(models.ReceiptValidation.objects.count(), 0)
 
 
+class ReceiptDataFetchTestCase(PopulatedLiveAfipTestCase):
+    def test_fetch_existing_data(self):
+        pos = models.PointOfSales.objects.first()
+        rt = models.ReceiptType.objects.get(code=6)
+        # last receipt number is needed for testing, it seems they flush old receipts
+        # so we can't use a fixed receipt number
+        last_receipt_number = models.Receipt.objects.fetch_last_receipt_number(pos, rt)
+        receipt = models.Receipt.objects.fetch_receipt_data(
+            receipt_type=6,
+            receipt_number=last_receipt_number,
+            point_of_sales=pos
+        )
+
+        self.assertEqual(receipt.CbteDesde, last_receipt_number)
```

Comment:
Con `pytest`, usar `assert` da output más amigable cuando falla el test por algún motivo:

```suggestion
        assert receipt.CbteDesde == last_receipt_number
```

Reply:
Listo, implementado en https://github.com/juanpsenn/django-afip/commit/a1c71cfef3fafda011b8fb46429b127fe849d968